scripts/common/rook.sh

```diff
@@ -216,7 +233,10 @@ function maybe_cleanup_rook() {
 
         if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then
             report_addon_start "rook-ceph-removal" "v1"
-            remove_rook_ceph
+            if ! remove_rook_ceph; then
+                logFail "Unable to remove Rook."
```

Comment:
@emosbaugh @laverya I do not think that we need to stop the upgrade because was not possible to remove Rook. IMO we just make clear that printing in red so that users are aware of and we have a chance to do the upgrade with success and afterwords remove rook manually.

Any objections to that?